lib/cloudregister/VERSION

```diff
@@ -1 +1 @@
-10.1.6
+10.1.7
```

Comment:
Don't need a version bump for this, we are not going to release just this change.